package.json

```diff
@@ -50,7 +50,14 @@
     "elliptic@6.5.4": "^6.5.7",
     "fast-xml-parser@^4.3.4": "^4.4.1",
     "send": "^0.19.0",
-    "ws@7.4.6": "^7.5.10"
+    "ws@7.4.6": "^7.5.10",
+    "minimist": "1.2.6",
+    "node-fetch": "^2.6.7",
+    "xmldom": "npm:@xmldom/xmldom@0.7.13",
+    "@walletconnect/socket-transport/ws": "^7.5.10",
+    "@ethersproject/providers/ws": "^7.5.10",
```

Comment:
It doesn't look like `@ethersproject/providers/ws` is in our dependency tree?

![Screenshot 2025-03-06 at 3 24 05 PM](https://github.com/user-attachments/assets/f6601fff-a951-4558-8b81-98517a398c07)